cosmo_tester/test_suites/cluster/full_cluster_test.py

```diff
@@ -402,6 +404,59 @@ def test_three_nodes_cluster_teardown(three_nodes_cluster, ssh_key,
     _assert_cluster_status(node1.client)
 
 
+def test_three_nodes_cluster_upgrade(three_nodes_5_1_0_cluster, ssh_key,
+                                     test_config, logger):
+    node1, node2, node3 = three_nodes_5_1_0_cluster
+    nodes_list = [node1, node2, node3]
+    logger.info('Installing example deployment')
+    example = get_example_deployment(node1, ssh_key, logger,
+                                     'cluster_teardown', test_config)
```

Comment:
I'm not convinced 'cluster_teardown' is the right name for this?

Reply:
You're absolutely right, I missed it.